cli/cli.hs

```diff
@@ -515,7 +515,9 @@ symvmFromCommand cmd calldata = do
                              )
 
   let
-    caller = SymAddr "caller"
+    caller = case cmd.caller of
+               Nothing -> SymAddr "caller"
+               Just a -> LitAddr a
```

Comment:
This is good, but to be consistent with the style of the code (see the next couple of lines) we could write the following?
```suggestion
    caller = maybe (SymAddr "caller") LitAddr cmd.caller
```